src/service-worker.ts

```diff
@@ -6,8 +6,8 @@ export {}
 
 // create-react-app calls workbox-webpack-plugin.InjectManifest by default
 // and will throw an error if __WEB_MANIFEST isn't referenced in the service worker file
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const manifest = self.__WB_MANIFEST
+// @ts-ignore
```

Comment:
What's that `@ts-ignore` for?

Reply:
This PR enforces the `noUnusedLocals` TypeScript rule - `manifest` is not used in this local file but is still required as per the comment above.

Reply:
Added additional commentary in https://github.com/shapeshift/web/pull/1664/commits/f0301905251d4a267af2583682f44e04cc9eef46

Reply:
Ok understood, seems a bit hacky

https://github.com/GoogleChrome/workbox/issues/2519#issuecomment-634164566

Not sure that we need this, but I don't know everything about the way we use service-worker there

Reply:
Yeah, that may well be a better way to handle the manifest (and I'm open to a new PR that changes it) - but this PR has only one purpose.

Reply:
The service worker is getting changed in an upcoming PR anyway